### PR TITLE
Bugfix view cherrypicked samples

### PR DESCRIPTION
**Pull request by emrojo**
Add biosero events to cherrypicked_samples view:

List of events: 
'lh_beckman_cp_destination_created', 'lh_biosero_cp_destination_plate_completed', 'lh_biosero_cp_destination_plate_partial_completed',
      'lh_biosero_cp_error_recovered_destination_plate_completed','lh_biosero_cp_error_recovered_destination_plate_partial_completed'

### CODE REVIEW

File: db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      IF(mlwh_events_event_types.key="lh_beckman_cp_destination_created","Beckman","Biosero") AS 'robot_type'
+      FROM #{mlwh_wh_db}.sample as mlwh_sample
+      JOIN #{mlwh_wh_db}.lighthouse_sample AS mlwh_lh_sample ON (mlwh_sample.uuid_sample_lims = mlwh_lh_sample.lh_sample_uuid)
+      JOIN #{event_wh_db}.subjects AS mlwh_events_subjects ON (mlwh_events_subjects.uuid = UNHEX(REPLACE(mlwh_lh_sample.lh_sample_uuid, '-', '')))
+      JOIN #{event_wh_db}.roles AS mlwh_events_roles ON (mlwh_events_roles.subject_id = mlwh_events_subjects.id)
+      JOIN #{event_wh_db}.events AS mlwh_events_events ON (mlwh_events_events.id = mlwh_events_roles.event_id)
+      JOIN #{event_wh_db}.event_types AS mlwh_events_event_types ON (mlwh_events_event_types.id = mlwh_events_events.event_type_id)
+      WHERE mlwh_events_event_types.key IN (
+        'lh_beckman_cp_destination_created', 'lh_biosero_cp_destination_plate_completed', 'lh_biosero_cp_destination_plate_partial_completed',
+        'lh_biosero_cp_error_recovered_destination_plate_completed','lh_biosero_cp_error_recovered_destination_plate_partial_completed'
```
Comment (KatyTaylor) [inline db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb:32]: So in all of these events, the samples have been 'cherrypicked'? They want all of these in the view?

File: db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# Enable again Biosero events in view
```
Comment (KatyTaylor) [inline db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb:3]: When it says "Enable **again**"... did the Biosero events used to be in the view?

File: db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb (hunk 1 above)
Comment (emrojo) [reply db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb:38]: Yes, they were before

File: db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb (hunk 2 above)
Comment (emrojo) [reply db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb:3]: Yes, we accidentally rolled back this in last update I think

Comment (KatyTaylor) [reply db/migrate/20211109125417_update_cherrypicked_samples_biosero.rb:3]: ah ok!

**Review by KatyTaylor — APPROVED** [review]

**Pull request merged**